model.yml

```diff
@@ -0,0 +1,21 @@
+description: empty
```

Comment:
Here by mistake, should be removed?

Reply:
You mean the whole file or just this one line?